test/test_multiprocessing.py

```diff
@@ -37,6 +38,29 @@ def run(self):
         self.tensor.add_(3)
 
 
+def _test_cuda_ipc_deadlock_actor(queue):
+    steps = 0
```

Comment:
why not loop over steps?
Also, better to send `steps` as argument instead of hardcoding it in 2 functions - they'll deadlock if number of steps is not the same